src/System.CommandLine.Subsystems.Tests/System.CommandLine.Subsystems.Tests.csproj

```diff
@@ -29,6 +29,8 @@
     <Compile Include="PipelineTests.cs" />
     <Compile Include="VersionFunctionalTests.cs" />
     <Compile Include="VersionSubsystemTests.cs" />
+    <Compile Include="ErrorReportingFunctionalTests.cs" />
+    <Compile Include="ErrorReportingSubsystemTests.cs" />
```

Comment:
Are these needed? I would have expected the default SDK to pick up *.cs files in the root on the project.

Reply:
We're currently using `<EnableDefaultCompileItems>False</EnableDefaultCompileItems>` to default exclude files and avoid compilation errors in files that aren't working yet. These will be ripped out later but needed at the moment